src/application/url/service.go

```
@@ -0,0 +1,73 @@
+package url
+
+import (
+	"crypto/md5"
```

golangci-lint (1.64.8)

[medium] G501: Blocklisted import crypto/md5: weak cryptographic primitive

(gosec)